src/adapters/postgres/fields-adapter.ts

```
@@ -2,304 +1,1773 @@
+
+            // If exist and isCreate is false, update the record
+            await this.updateSourceDetailsTableFields(
+              getSourceDetails.sourceDetails.table,
+              sourceFieldName["name"],
+              sourceFieldName["value"],
+              updatedBy,
+              sourceFieldName["controllingfieldfk"]
+            );
+          }
         }
+        delete fieldsData.fieldParams;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)